river/drift/hddm_a.py

```diff
@@ -6,11 +6,19 @@
 class HDDM_A(DriftDetector):
     """Drift Detection Method based on Hoeffding’s bounds with moving average-test.
 
-    HDDM_A is a drift detection method based on the Hoeffding’s inequality. HDDM_A uses
-    the average as estimator. It receives as input a stream of real values and
-    returns the estimated status of the stream: STABLE, WARNING or DRIFT.
+    HDDM_A is a drift detection method based on the Hoeffding’s inequality.
+    HDDM_A uses the average as estimator. It receives as input a stream of real
+    values and returns the estimated status of the stream: STABLE, WARNING or
+    DRIFT.
 
-    Implementation based on MOA.
+    **Input:** HDDM_A expects `value` to be a binary value as input, indicating
+     whether the last sample analyzed was correctly classified.
+
+    - 0: Correclty classifies
```

Comment:
Typo in "Correctly"

Reply:
Reworded this comment for clarity in 27ef120bd2b0d3c4e1560e223db91ffba0415b16

Same applies to `DDM`, `EDDM`, `HDDM_A`, and `HDDM_W`.